main.go

```diff
@@ -53,6 +53,14 @@ func main() {
 	if err != nil {
 		log.WithField("Error", err).Fatal("Could not fetch config options for etcd.")
 	}
+	oldCfg, err := etcdmgmt.ReadEtcdConfig()
```

Comment:
With changes done to `ReadEtcdConfig`, this becomes simpler `if cannot read create new`.